sandbox

```diff
@@ -534,8 +607,10 @@ sandbox () {
       wait_for_services
       echo "* services ready!"
 
+      set +e
       version_helper
       status_helper
+      set -e
```

Comment:
To get here all of the services had already started properly, so no need to exit. Things are a bit slower on CI and these were generating a lot of false negatives.